packages/mermaid/src/docs/syntax/entityRelationshipDiagram.md

```diff
@@ -409,17 +409,33 @@ erDiagram
 
 ### Renderer
 
-The layout of the diagram is done with the renderer. The default renderer is dagre.
+The layout of the diagram is handled by [`render()`](../config/setup/mermaid/interfaces/Mermaid.md#render). The default layout is dagre.
 
-You can opt to use an alternate renderer named elk by editing the configuration. The elk renderer is better for larger and/or more complex diagrams.
+For larger or more-complex diagrams, you can alternatively apply the ELK (Eclipse Layout Kernel) layout using your YAML frontmatter's `config`. For more information, see [Customizing ELK Layout](../intro/syntax-reference.md).
 
 ```
 ---
-    config:
-        layout: elk
+config:
+    layout: elk
```

Comment:
Our linter is complaining about the indentation, but using `npx prettier --write packages/mermaid/src/docs/syntax/entityRelationshipDiagram.md` fixes this.

```suggestion
  layout: elk
```

Reply:
Fixed in https://github.com/mermaid-js/mermaid/pull/6426/commits/de72e18a7fa0894f9eac5c60670299dfc7a533fc